kebechet/managers/update/update.py

```diff
@@ -493,9 +497,27 @@ def _relock_all(self, exc: PipenvError, labels: list) -> None:
         self._pipenv_update_all()
         commit_msg = "Automatic dependency re-locking"
         branch_name = "kebechet-dependency-relock"
-        self._git_push(":pushpin: " + commit_msg, branch_name, ['Pipfile.lock'])
-        pr_id = self.sm.open_merge_request(commit_msg, branch_name, f"Fixes: #{issue.id}", labels)
-        _LOGGER.info(f"Issued automatic dependency re-locking in PR #{pr_id} to fix issue #{issue.id}")
+
+        existing_prs = self._get_prs(branch_name)
+        if len(existing_prs) == 1:
+            pr = list(existing_prs)[0]
+            commits = pr.get_all_commits()
+            if len(commits) != 1:
+                _LOGGER.info("There have been done changes in the original pull request (multiple commits found), "
+                             "aborting doing changes to the adjusted opened pull request")
```

Comment:
Will this be valuable for the user on GitHub? (Do we want to notify the user with a comment?)

Reply:
Might be useful, I will add it.